tech/tech.py

```diff
@@ -95,65 +55,146 @@ def __init__(self, total_impact_hours, impact_hour_data, total_cstk_tokens,
         self.param.maximum_impact_hour_rate.step = config['maximum_impact_hour_rate']['step']
         self.maximum_impact_hour_rate = config['maximum_impact_hour_rate']['value']
 
-        #self.param.hatch_oracle_ratio.bounds = config['hatch_oracle_ratio']['bounds']
-        #self.param.hatch_oracle_ratio.step = config['hatch_oracle_ratio']['step']
         self.hatch_oracle_ratio = config['hatch_oracle_ratio']['value']
-
-        #self.param.hatch_period_days.bounds = config['hatch_period_days']['bounds']
-        #self.param.hatch_period_days.step = config['hatch_period_days']['step']
         self.hatch_period_days = config['hatch_period_days']['value']
-
-        #self.param.hatch_exchange_rate.bounds = config['hatch_exchange_rate']['bounds']
-        #self.param.hatch_exchange_rate.step = config['hatch_exchange_rate']['step']
         self.hatch_exchange_rate = config['hatch_exchange_rate']['value']
-
-        #self.param.hatch_tribute_percentage.bounds = config['hatch_tribute_percentage']['bounds']
-        #self.param.hatch_tribute_percentage.step = config['hatch_tribute_percentage']['step']
         self.hatch_tribute_percentage = config['hatch_tribute_percentage']['value']
 
-    @param.depends('action')
-    def payout_view(self):
-        scenario_rates = self.get_rate_scenarios()
-        self.impact_hour_data['Assumed Target Goal (wxDai)'] = (self.impact_hour_data['Assumed IH'] * scenario_rates['target_rate']).round(2)
-        self.impact_hour_data['Assumed Min Goal (wxDai)'] = (self.impact_hour_data['Assumed IH'] * scenario_rates['min_rate']).round(2)
-        self.impact_hour_data['Assumed Max Goal (wxDai)'] = (self.impact_hour_data['Assumed IH'] * scenario_rates['max_rate']).round(2)
 
-        self.impact_hour_data['Min IH Target Goal (wxDai)'] = (self.impact_hour_data['Min IH'] * scenario_rates['target_rate']).round(2)
-        self.impact_hour_data['Min IH Min Goal (wxDai)'] = (self.impact_hour_data['Min IH'] * scenario_rates['min_rate']).round(2)
-        self.impact_hour_data['Min IH Max Goal (wxDai)'] = (self.impact_hour_data['Min IH'] * scenario_rates['max_rate']).round(2)
+    # Utils
+    def get_overview_data(self):
```

Comment:
this function uses other functions declared below, like `get_overview_data` and  `get_impact_hour_rate`. it would be better to declare `get_overview_data` after them.